### PR TITLE
Remove all stale pages and revise what's left.

### PR DESCRIPTION
**Pull request by vext01**

### CODE REVIEW

File: src/intro.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-Yorick is a fork of the Rust programming language which aims to build a
-meta-tracing system using Rust as a meta-language. Imagine PyPy, but where your
-interpreter is written in Rust instead of RPython.
+yk is a meta-tracing system built on LLVM.
 
 Note that the system is in early stages of development: don't expect it to work
 yet.
 
-Yorick is developed by [The Software Development Team](https://soft-dev.org/)
+yk is developed by [The Software Development Team](https://soft-dev.org/)
```
Comment (ltratt) [inline src/intro.md:8]: I think we should delete this sentence: we're the initial developers, but we don't want people to feel they can't contribute. Thoughts?

Comment (vext01) [reply src/intro.md:8]: OK by me.

Comment (ltratt) [reply src/intro.md:8]: OK, off with its head :)

**Comment by vext01** [comment]
OK to squash?

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Comment by vext01** [comment]
Hold on, something is wrong.

**Comment by vext01** [comment]
There we go. Sorry about that.

**Comment by ltratt** [comment]
What changed?

**Comment by vext01** [comment]
I think I branched off of a not up to date master branch, so there was a conflict. I've resolved the conflict and checked it all still looks good.

**Comment by vext01** [comment]
[At some point we should move this repo to ykjit org]

**Pull request merged**